spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1,5 +1,8925 @@
+:sectnums!:
+== Preface
+
+This is the Jakarta Server Pages™ Standard Tag
+Library 2.0 (JSTL 2.0) specification, developed by the JSR-52 expert
```

Comment:
I think the reference to the JSR-52 needs to be removed here.

Reply:
Also do you think we should change `Jakarta Server Pages Standard Tag Library 2.0` to just `Jakarta Standard Tab Library` since that more accurately reflects the project name: https://jakarta.ee/specifications/tags/

Reply:
Yes, definitely! :)